bindings/rust/bench/README.md

```diff
@@ -32,6 +32,8 @@ To bench with AWS-LC, Amazon's custom libcrypto implementation, first run `scrip
 
 The benchmarks can be run with the `cargo bench` command. Criterion will auto-generate an HTML report in `target/criterion/`. 
 
+Throughput benchmarks measure round-trip throughput with the client and server connections in the same thread for symmetry. In practice, throughput for a single connection would be ~4x higher than the values from the benchmarks.
```

Comment:
I don't think I totally understand this statements? Wouldn't it just be 2x higher? 

I'm also wary of telling people what their numbers "might" look like. In fact, it might be a good idea to have a general disclaimer about how we abstract away a lot of the complexity that exists in real systems(network IO) so customers should always run real benchmarks themselves, etc.

Reply:
Throughput being one-way vs round-trip makes it 2x higher, and having send/recv be in the same thread instead of two different threads makes it 2x higher on top of that, multiplying to 4x.

Putting a general disclaimer in an obvious place seems to be a good idea; I'll add it at the top.